exercises/luhn/luhn_test.exs

```diff
@@ -0,0 +1,44 @@
+if !System.get_env("EXERCISM_TEST_EXAMPLES") do
+  Code.load_file("luhn.exs")
+end
+
+ExUnit.start
+ExUnit.configure exclude: :pending, trace: true
+
+defmodule LuhnTest do
+  use ExUnit.Case
+
+  test "checksum" do
+    assert Luhn.checksum("4913") == 22
+  end
+
+  @tag :pending
+  test "checksum again" do
```

Comment:
I'm a fan of having a single case per `test`, which is typically equal to having 1 single assert per test, and see no improvement to compact them at all other than just to have them compacted

I know I've caught myself with errors on the `again` test from some exercisms that never showed up on the first test